controls/radpdfviewer/ui/find-dialog.md

```diff
@@ -1,91 +1,99 @@
-		ExtensibilityManager.RegisterFindDialog(new FindDialog());
-{{endregion}}
-
-
-{% if site.site_name == 'WPF' %}        
-If you prefer adding the dialog through XAML, you can use the RegisterFindDialog property of the RadPdfViewerAttachedComponents class.
-
-#### [XAML] Example 1: Register FindDialog in XAML 
-
-{{region radpdfviewer-find-dialog_1}}
-		<telerik:RadPdfViewer telerik:RadPdfViewerAttachedComponents.RegisterFindDialog="True" />
-
-{{endregion}}
-
-{% endif %}
-
-
-The __ShowFindDialogCommand__ handles the opening of the find dialog. This command is bound to the __Ctrl+F__ shortcut key combination (__Apple+F__ for Mac) in RadPdfViewer.
-        
-
-## Creating a custom find dialog
-
-RadPdfViewer offers an easy way to create a custom find dialog which suits best to different user’s needs and desires.
-
-
-In order to achieve this, should be created a class which inherits the __IFindDialog__ interface with its __ShowDialog__ method.
-        
-
-#### [C#] Example 2:Create custom find dialog
-
-{{region radpdfviewer-find-dialog_2}}
-	    public class CustomFindDialog : Window, IFindDialog
-	    {
-	        public void ShowDialog(FindDialogContext context)
-	        {
-	            // ...
-	        }
-	    }
-{{endregion}}
-
-
-
->tipFind a runnable project example demonstrating this functionality in our online SDK repository [here](https://github.com/telerik/xaml-sdk), the example is listed as __PdfViewer / CustomFindDialog__.
-          
+---
+title: Find Dialog
+page_title: Find Dialog
+description: Find Dialog
+slug: radpdfviewer-find-dialog
+tags: find,dialog
+published: True
+position: 4
+---
+
+# Find Dialog
+
+
+__RadPdfViewer__ allows flexible searching in a loaded PDF document. The following article describes the available search options, as well as how to use and customize the find dialog.
+      
+
+>importantOther than the default needed [assemblies]({%slug radpdfviewer-getting-started%}), in order to take advantage of the Find dialog functionality in __RadPdfViewer__ you need to add to your project a reference to __Telerik.Windows.Controls.Navigation.dll__.
+        
+
+## Using the UI
+
+There are three possible input parameters that determine the results of the search:
+        
+
+1. Regular Expression: Finds and returns entry of specified regular expression.
+            
+
+1. Case Sensitive: Finds and returns all entries of a specified string taking into consideration case-sensitive. For example, you are going to find “AB” string, strings such as “aB”, “Ab” and “ab” would be ignored in the results.
+            
+
+1. Whole Words Only: Finds and returns all entries of a given string which should be a whole word only (not a part of a word).
+            
+
+This is how the Find Dialog looks:
+![Rad Pdf Viewer Find Dialog](images/RadPdfViewer_Find_Dialog.png)
+
+## Find Programmatically
+
+Find Programmatically is described in details in [Find]({%slug radpdfviewer-find%}) article.
+
```

Comment:
The article title is Find Dialog which suggests that the dialog is described inside, so we don't need a specific section for the programmatic search here. Add it as a note in case someone has found it in searching for the API.